inveniosoftware-contrib/intbitset
# Support Python 3.12 and drop 3.7

This PR uses the latest Cython and refines the build to:
1. add support for Python 3.12
2. drop test run on Ubuntu 18 and older macOS
3. add support for newest macOS
4. drop support for Python 3.7

This PR should be fixing these build issues:
- fix for #81
- fix for #89
- fix for #95
